app/src/main/AndroidManifest.xml

```diff
@@ -90,6 +90,16 @@
             android:theme="@style/Theme.Crf.Onboarding"
             android:windowSoftInputMode="adjustResize" />
 
+        <provider
```

Comment:
I developed an app earlier in the year where I attempted to use FileProvider to share images with other apps and what I found was that there is a serious segmentation problem with using FileProvider.  It seemed pretty random with various devices/platforms/apps that didn't support it.  Granted, I was using it to communicate with many apps and 3rd party apps, but I even found that default android messaging apps didn't support it on various devices.  Do you have any evidence that all gallery and camera intents now support this protocol of sharing content?

Reply:
If you target sdk >= 24, you need to use a FileProvider here otherwise you will get a security exception and the camera app cannot write to the file you provide.  I originally wrote the code without the FileProvider but had to change it.  I am not sure if older devices will have an issue.  I can try it on a few older devices that I have.

Reply:
Are you sure the security exception wasn't from you not enabling the read/write external storage permission?

Reply:
Yes, I am not writing to external storage. https://stackoverflow.com/questions/38200282/android-os-fileuriexposedexception-file-storage-emulated-0-test-txt-exposed